hrm-domain/lambdas/custom-integrations/uscr-beacon-poller/tests/unit/caseReport.test.ts

```diff
@@ -0,0 +1,457 @@
+/**
```

Comment:
Double checking that our audit logging will continue to work just fine for editing cases from this workflow, yes? Both the internal database audit logs, and the "Updated by" that shows in the UI?

Reply:
Yes, it will show as 'updated by: system' in the UI but otherwise it will work as expected in both cases